docs/source/config.rst

```diff
@@ -0,0 +1,64 @@
+Configuring Jupyter applications
+================================
+
+The Jupyter applications have a common config system.
+
+.. _config_dir:
+
+The Jupyter config directory
+----------------------------
+
+Config files are stored in :file:`~/.jupyter` by default.
```

Comment:
Yeah, we use different directories for different things, but the config files we expect users to edit are in `~/.jupyter` on all platforms, and that's what this is about.